src/ess/reduce/live/raw.py

```diff
@@ -455,9 +528,12 @@ def position_noise_for_cylindrical_pixel(
 
 
 def gaussian_position_noise(sigma: PositionNoiseSigma) -> PositionNoise:
+    sigma = sigma.to(unit='m', copy=False)
     size = _noise_size
     position = sc.empty(sizes={'position': size}, unit='m', dtype=sc.DType.vector3)
-    position.values = np.random.default_rng().normal(0, sigma.value, size=(size, 3))
+    position.values = np.random.default_rng(seed=1234).normal(
```

Comment:
Is this to turn the centers of the pixel positions into small gaussian clouds of points?
I was thinking maybe it should be `random.uniform` instead to avoid spill-over from one pixel over to the next?

But maybe using a normal distribution is better as it makes the clouds spherical, and they then look the same from any line-of-sight? Using uniform also suggests that the pixels are square in shape, but this is not really true for tubes...

Is the sigma approximately equal to half the width of the pixel?

Reply:
The gaussian option is just a simple alternative to something that uses the pixel shape, see the cylinder case (used for Loki tubes). We may eventually have something similar for other shapes, I think.

Sigma is a parameter than can be set in a config file in Beamlime.